Dockerfile/Dockerfile.Nginx

```diff
@@ -0,0 +1,19 @@
+# node 20, build the application
+FROM node:20.12.2-alpine3.19
+COPY ../Interface .
```

Comment:
I do not have the Interface folder. Where does it come from?

is it nextjs-interface ?